lib/index.js

```diff
@@ -263,7 +273,7 @@ class Service extends _adapterCommons.AdapterService {
         value = JSON.parse(value);
       }
 
-      return operator === '=' ? query.where(column, value) : query.where(column, operator, value);
+      return operator === '=' ? query.where(column, value) : query.where(CAST_TO_TEXT.includes(operator) ? this.Model.raw(`"${column}"::text`) : column, operator, value);
```

Comment:
Where you have used .raw can you use the parameters so that ${column} is escaped by Knex to help prevent against sql injection? This is the example from creating indexes

`knex.raw(``CREATE`` INDEX IF NOT EXISTS ?? ON ?? ( ?? )``, [
                    ``ix_${model.collectionName.toLowerCase()}_${indexProperty.toLowerCase()}``,`